lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1417,21 +1418,80 @@ func (w *WalletKit) fundPsbtInternalWallet(account string,
 				return err
 			}
 
+			// filterFn makes sure utxos which are unconfirmed and
+			// still used by the sweeper are not used.
+			filterFn := func(u *lnwallet.Utxo) bool {
```

Comment:
same question as prev commit - could we not expand `ListUnspentWitness` to take the UtxoFilter functional option?